Signal/src/ViewControllers/ConversationView/ConversationViewController.m

```diff
@@ -2248,12 +2234,11 @@ - (void)ensureScrollDownButton
     // one page.
     BOOL isScrolledUp = scrollSpaceToBottom > pageHeight * 1.f;
 
-    if (numberOfMessages > 0) {
-        TSInteraction *lastInteraction =
-            [self interactionAtIndexPath:[NSIndexPath indexPathForRow:(NSInteger)numberOfMessages - 1 inSection:0]];
```

Comment:
Similarly, we want to stop using `interactionAtIndexPath` in favor of the view items.